File: src/main/kotlin/org/pocketretail/core/deliverylayer/rest/model/DeliveryLayerErrorResponse.java
package org.pocketretail.core.deliverylayer.rest.model;

import java.util.Date;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class DeliveryLayerErrorResponse extends DeliveryLayerCommonResponse{


    private String stackTrace;

    public DeliveryLayerErrorResponse(String msg, Integer status, Date timestamp,
                                      String stackTrace) {
        super(msg, status, timestamp);
        this.stackTrace = stackTrace;
    }
}

File: src/main/kotlin/org/pocketretail/core/deliverylayer/rest/model/DeliveryLayerErrorResponse.kt
package org.pocketretail.core.deliverylayer.rest.model

import java.util.Date

class DeliveryLayerErrorResponse(
    msg: String,
    status: Int,
    timestamp: Date,
    var stackTrace: String
) : DeliveryLayerCommonResponse(msg, status, timestamp)
